app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource
```

Comment:
## Dear second reviewer: We already have load_and_authorize_resource here in the posts_controller so no changes as been made!!